src/sentry/ingest/test.json

```diff
@@ -0,0 +1,3 @@
+{"event_id":"65fad5091a3f4db1b9812777f6dcc052","sent_at":"2023-02-28T16:05:58.058Z","sdk":{"name":"sentry.javascript.nextjs","version":"7.39.0"}}
```

Comment:
why do we need this file?

Reply:
Yep... I think it was included by mistake @priscilawebdev  ?

Reply:
yes that was included by mistake... I am removing and turning this back to "ready to review"  😉